dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetFieldType.java

```diff
@@ -520,7 +522,7 @@ public int compareTo(DatasetFieldType o) {
 
     public String getDisplayName() {
         if (isHasParent() && !parentDatasetFieldType.getTitle().equals(title)) {
-            return parentDatasetFieldType.getLocaleTitle() + " " + getLocaleTitle();
+            return Optional.ofNullable(getLocaleTitleWithParent()).filter(s -> !s.isEmpty()).orElse(parentDatasetFieldType.getLocaleTitle() + " " + getLocaleTitle());
```

Comment:
Was there a bug before with this?

Reply:
There was no bug, but now logic changes. If there is a translation for field with parent, then we get that. Else we use old logic of concatenating parent + child values (for example for now @wfenrich didn't make english translations for fields with parents)